back/src/whombat/models/annotation_task.py

```diff
@@ -100,12 +100,14 @@ class AnnotationTask(Base):
     clip: orm.Mapped[Clip] = orm.relationship(
         init=False,
         repr=False,
+        lazy="joined",
```

Comment:
Eagerly joining the Clip and ClipAnnotation tables in every Task query can introduce unnecessary performance overhead, especially since these tables already join with Recording and annotation data. This can lead to slower query response times and unnecessary data retrieval.

To optimize, let's avoid automatic joins at the table level and instead use explicit joins only when Clip or ClipAnnotation data is required for a specific Task query. This keeps the base Task query lightweight and improves performance in scenarios where quick task loading is crucial (e.g., during annotation).